meta-flutter/recipes-flutter/embedded-linux/flutter-client_git.bb

```diff
@@ -0,0 +1,19 @@
+SUMMARY = "Wayland backend (Stand-alone Wayland app)"
```

Comment:
Flutter embedding for embedded Linux with Wayland backend

Reply:
I fixed it.